stable/database/files/nuobackup

```diff
@@ -42,6 +44,64 @@ labels=""
 semaphore=""
 timeout=1800
 backup_root=$BACKUP_DIR
+wait_for_num_processes=0
+wait_timeout=300
```

Comment:
is there potentially any collision with K8s max job durations?

Reply:
I suppose that you are referring to `.spec.activeDeadlineSeconds` parameter?
We don't set this for backup jobs.